lepidopter-fh/etc/logrotate.d/ooniprobe

```diff
@@ -1,10 +1,9 @@
 /var/log/ooni/*.log {
-    weekly
     missingok
-    rotate 2
+    rotate 1
     compress
     delaycompress
     copytruncate
-    maxsize 10M
+    maxsize 1M
     notifempty
```

Comment:
I think for the `cronjobs.log` it's ok as is.

Can we have 1 entry for every time of log?